modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DELinkToPageField.testcase

```diff
@@ -81,12 +81,45 @@ definition {
 	}
 
 	@description = "This is a stubbed description"
-	@ignore = "true"
 	@priority = "5"
 	test CanClearInput {
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Test Page Name");
```

Comment:
@guivilarim, it's a best practice when writing a test to remove the necessary resources created to a test run, like this page `Test Page Name`, after all assertions of the test being created. Is this page being deleted after this test run? If not, we should do it.